docs/contribute.rst

```diff
@@ -135,6 +135,13 @@ Run selected tests or plans in verbose mode:
     tmt run --verbose plan --name basic
     tmt run -v test -n smoke
 
+You might want to set some useful environment variables when
+working on ``tmt`` tests, for example ``TMT_FEELING_SAFE`` to
+allow the ``local`` provision method or ``TMT_SHOW_TRACEBACK`` to
+show the full details for all failures. Consider installing the
+``direnv`` command which can take care of these for you.
```

Comment:
Installing just the package is afaik insufficient, you need to hook it to your shell, maybe we could point the user to the official docs?

https://direnv.net/#basic-installation

Reply:
Yeah, makes sense. Added the link in 2687d4de.